src/components/DistanceEReceipt.tsx

```diff
@@ -95,9 +87,9 @@ function DistanceEReceipt({transaction}) {
                                     style={styles.gap1}
                                     key={key}
                                 >
-                                    <Text style={styles.eReceiptWaypointTitle}>{translate(descriptionKey)}</Text>
-                                    {waypoint.name && <Text style={styles.eReceiptWaypointAddress}>{waypoint.name}</Text>}
-                                    {waypoint.address && <Text style={styles.textLabelSupporting}>{waypoint.address}</Text>}
+                                    <Text style={styles.eReceiptWaypointTitle}>{translate(descriptionKey as TranslationPaths)}</Text>
+                                    {waypoint?.name && <Text style={styles.eReceiptWaypointAddress}>{waypoint.name}</Text>}
+                                    {waypoint?.address && <Text style={styles.textLabelSupporting}>{waypoint.address}</Text>}
```

Comment:
Otherwise if the `waypoint?.name` or `waypoint?.address` are empty strings it could trigger a warning on the web platform about rendering string outside of the Text component
```suggestion
                                    {!!waypoint?.name && <Text style={styles.eReceiptWaypointAddress}>{waypoint.name}</Text>}
                                    {!!waypoint?.address && <Text style={styles.textLabelSupporting}>{waypoint.address}</Text>}
```